gradle/maven-central.gradle

```diff
@@ -34,4 +34,12 @@ project.ext.configureMavenCentralMetadata = {
     root.appendNode('description', 'Coroutines support libraries for Kotlin')
     root.appendNode('url', 'https://github.com/Kotlin/kotlinx.coroutines')
     root.children().last() + pomConfig
+
+    def dependenciesNode = asNode().getAt("dependencies")[0]
+    if (dependenciesNode != null) {
+        def dependencyNode = dependenciesNode.appendNode('dependency')
+        dependencyNode.appendNode('groupId', "$group")
+        dependencyNode.appendNode('artifactId', "$project.name")
+        dependencyNode.appendNode('version', "$project.version")
```

Comment:
What's the reason for the above section? What I see in the resulting .pom files:
* `kotlinx-coroutines-core` (JVM) artifact depends on _itself_
* `kotlinx-coroutines-core-js` artifact depends on `kotlinx-coroutines-core` (JVM)... very strange
* `kotlinx-coroutines-core-common` artifact depends on `kotlinx-coroutines-core` (JVM)... even stranger.
Something is not right here.